index.js

```diff
@@ -0,0 +1,85 @@
+let bookArray = [];
```

Comment:
- [ ] As pointed in the requirement screenshotted below, arrays should be inside the class, kindly refactor the code to do that
![image](https://user-images.githubusercontent.com/73140461/148272163-fe941b48-2f9a-44b9-8915-1f3d8b5bea29.png)